client/src/component/ticketForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import axios from 'axios';
+
+
+const TicketForm = () => {
+    //useState
+    const [firstName, setFirstName] = useState('')
+    const [lastName, setLastName] = useState('')
+    const [phone, setPhone] = useState('')
+    const [pickUpDate, setPickUpDate] = useState('')
+    const [ticketNumber, setTicketNumber] = useState('')
+    const [dropDate, setDropDate] = useState('')
+    // const handleChange = (event) => {
+    //     setFirstName(event.target.value);
+    //     console.log(event.target);
+    //   };
+
+    //Axios Put request 
+    const callApi = () => {
+        //Endpoint to express server 
+        const url = 'http://localhost:5000/testAPI';
+        
+        const ticketData = {
+            firstName, lastName, phone, pickUpDate, ticketNumber, dropDate
+        }
+        console.log(ticketData);
+
+        //Use post or put request?
+        axios.put(url, ticketData)
```

Comment:
https://www.w3schools.com/tags/ref_httpmethods.asp#:~:text=PUT%20is%20used%20to%20send,that%20PUT%20requests%20are%20idempotent.&text=In%20contrast%2C%20calling%20a%20POST,the%20same%20resource%20multiple%20times.

So this technically depends on how we build the API.